.github/workflows/surge.yml

```
@@ -0,0 +1,36 @@
+
+# See https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions
+
+name: Surge
+
+on:
+  push:
+    branches:
+      # publish only the latest changes to master
+      - master
+
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    container: registry.opensuse.org/yast/head/containers/yast-cpp:latest
+
+    steps:
+
+    - name: Git Checkout
+      uses: actions/checkout@v1
+
+    - name: Prepare
+      run: |
+        zypper  in -y npm doxygen
+        npm install --global surge
+
+    - name: Generate Doc
+      run: rake doc
+
+    - name: Surge
+      run:  surge --project ./autodocs/html --domain yast-ui-bindings.surge.sh
```

Oops, wrong domain...
```suggestion
      run:  surge --project ./autodocs/html --domain yast-pkg-bindings.surge.sh
```

---

good catch. Copy and paste error. Luckily it won't submit, as credentials are for correct project